docs/datamodel/props.rst

```diff
@@ -34,3 +33,13 @@ source objects can link to the same target via
 out the number of property targets, a :ref:`collection type
 <ref_datamodel_collection_types>` target by itself is considered a
 *single* target.
+
+
+See Also
+--------
+
+Link
```

Comment:
Is "link" supposed to be here? Or is it "property"?

Reply:
Fixed